nsv13/code/modules/food_and_drinks/food/custom_food.dm

```diff
@@ -0,0 +1,11 @@
+/obj/item/reagent_containers/food/snacks/hardtack //NSV13
```

Comment:
You don't need to keep the NSV tags when it's already in a custom file, and you need to include these files in the nsv13.dme file as well ("ticking" it in dream maker or VSC)